src/praisonai/tests/unit/integrations/test_managed_agents.py

```
@@ -0,0 +1,197 @@
+            self.executed_prompts.append(prompt)
+            self.execution_kwargs.append(kwargs)
+            yield {
+                'type': 'agent.message',
+                'content': [{'type': 'text', 'text': f"Backend streamed: {prompt}"}]
+            }
+    
+    # Create mock backend
+    mock_backend = MockManagedBackend()
+    
+    # Create agent with backend
+    agent = Agent(
```

Ruff (0.15.9)

[error] Undefined name `Agent`

(F821)